src/Analysis/Ast/Impl/Analyzer/Handlers/ImportHandler.cs

```diff
@@ -56,7 +56,7 @@ internal sealed partial class ImportHandler : StatementHandler {
             // import_module('fob.oar.baz')
             var importNames = ImmutableArray<string>.Empty;
             var lastModule = default(PythonVariableModule);
-            var modules = new PythonVariableModule[moduleImportExpression.Names.Count];
+            var modules = new (string, PythonVariableModule)[moduleImportExpression.Names.Count];
```

Comment:
You should actually be able to give the tuple items names, like `string name, PythonVariableModule module` and avoid the `Item1`/`Item2` thing below.

Reply:
Done